modules/core/src/views/types.ts

```diff
@@ -0,0 +1,19 @@
+import type {MapViewState} from './map-view';
+import type {FirstPersonViewState} from './first-person-view';
+import type {OrbitViewState} from './orbit-view';
+import type {OrthographicViewState} from './orthographic-view';
+import type {GlobeViewState} from './globe-view';
+
+export type AnyViewState =
```

Comment:
Deck supports custom Views and ViewState beyond these types, right? Is there a way to extend the type, perhaps with a generic or an override?